### ISSUE TITLE
ModuleNotFoundError: No module named 'liblinearutil'

### ISSUE THREAD

**Issue by sophiejiaojiao** [yangysc/document-classification#1]
Hello，I cloned your code and tried to run it. I installed Pandas and Liblinear, but always get "ModuleNotFoundError: No module named 'liblinearutil'". 
I guess that I have not successfully installed Liblinear.  So, could you help me and tell me how to solve this problem?
Thank you.

**Comment by yangysc** [comment]
Hi,
I guess that your OS is Windows, please install Liblinear provided by this website [liblinear](https://www.lfd.uci.edu/~gohlke/pythonlibs/#liblinear), then open a cmd terminal, running the command below to install it locally.
`pip install ./liblinear‑2.11‑cp36‑cp36m‑win_amd64.whl`
where the version of .whl depends on your python version.

Closing this issue, please reopen it if you have any more questions.

**Comment by sophiejiaojiao** [comment]
Hi,
Thanks for your reply of my issue#1. I installed the Liblinear following your method, but got the error message again.
My OS is Windows10 and python version is 3.6. I put three screenshot here:
The first following screenshot shows that I successfully installed liblinear (liblinear-2.11-cp36-cp36m-win_amd64.whl).
The second shows the error message and the third one shows the exact location of the error message.
![1](https://user-images.githubusercontent.com/22901682/34432817-cdf453c2-ecb5-11e7-92b7-026cc69b6154.png)
![2](https://user-images.githubusercontent.com/22901682/34432824-db9ea5d6-ecb5-11e7-99d3-1f31e1098272.png)
![3](https://user-images.githubusercontent.com/22901682/34432826-deafeb18-ecb5-11e7-97ee-f2fdaeb9056e.png)
